graylog2-server/src/main/java/org/graylog/events/processor/systemnotification/SystemNotificationEventsIndexMapping7.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.events.processor.systemnotification;
+
+import com.google.common.collect.ImmutableMap;
+import org.graylog2.indexer.EventsIndexMapping7;
+
+public class SystemNotificationEventsIndexMapping7 extends EventsIndexMapping7 {
```

Comment:
```suggestion
public class SystemEventsIndexMapping extends EventsIndexMapping7 {
```
I would drop the `Notification` from the name, because it will be used for system events.
Also the `7` is unnecessary, because we don't support ES 6 anymore.

Reply:
done